<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyUserInfoDialog</class>
 <widget class="QDialog" name="modifyUserInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#modifyUserInfoDialog{
border-image:url(:/icon/icon/kobe2.jpg);
}
#modifyUserInfoDialog *{
border-image:url();
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>原密码  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="oldPasswordLineEdit">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 135, 0);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>新密码  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPasswordLineEdit">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 135, 0);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPassword2LineEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 135, 0);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>取消/返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyPasswordButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>确认密码修改</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyNameButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>确认昵称修改</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>昵称    </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 135, 0);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
